main/createUiDefinition.json

```diff
@@ -139,7 +151,7 @@
                 "label": "Subnet",
                 "defaultValue": {
                   "name": "subnet1",
-                  "addressPrefixSize": "/24"
+                  "addressPrefixSize": ""
```

Comment:
pls remove addressPrefixSize field as not needed more.

Reply:
done